docs/python_gpu.rst

```diff
@@ -6,6 +6,9 @@ Quick-start examples
 
 As mentioned in the :ref:`Python GPU installation instructions <install-python-gpu>`, the easiest way to install the Python interface for cuFINUFFT is to run ``pip install cufinufft``.
```

Comment:
Does pip install cufinufft install legacy 1.3 or just the GPU part of 2.2? I'm actually unclear...

Reply:
Currently, it installs legacy 1.3. Once we've done the full release (currently we have only published beta wheels), it'll install the GPU part of 2.2 (AKA cufinufft 2.2).